pkg/shares/share_splitting.go

```diff
@@ -72,21 +72,22 @@ func SplitTxs(txs coretypes.Txs) (txShares []Share, pfbShares []Share, shareRang
 	return txShares, pfbShares, mergeMaps(txMap, pfbMap), nil
 }
 
-func SplitBlobs(cursor int, indexes []uint32, blobs []coretypes.Blob, useShareIndexes bool) ([]Share, error) {
-	if useShareIndexes && len(indexes) != len(blobs) {
-		return nil, ErrIncorrectNumberOfIndexes
+// SplitBlob splits the provided blob into shares.
+func SplitBlob(blob coretypes.Blob) ([]Share, error) {
+	writer := NewSparseShareSplitter()
+	if err := writer.Write(blob); err != nil {
+		return nil, err
 	}
+	return writer.Export(), nil
+}
+
+// SplitBlobs splits the provided blobs into shares.
```

Comment:
```suggestion
// SplitBlobs splits the provided blobs into shares belonging to the same sequence. All blobs should thus belong to the same namespace
```
I think it's important to specify that they should be of the same namespace. Does the `SparseShareSplitter` already panic if shares are from different namespaces?

Reply:
`SplitBlobs` can be called with blobs from different namespaces. SparseShareSplitter supports writing blobs from different namespaces.

Going to revert this comment.

Reply:
Resolved by 245d2431d57eb7ff310ff3814d9fc41cfc9055fc